pyproject.toml

```diff
@@ -0,0 +1,115 @@
+[project]
+name = "azure-functions-worker"
+version = "4.29.0"
```

Comment:
We can make the version dynamic and pull it from azure_functions_worker/version.py